### PR TITLE
Translation: redis.adoc.po

### PR DESCRIPTION
**Pull request by judalabs**
Here is the Portuguese translation for the redis.adoc.po file.
I translated the '_Increment_' code as well
That PR is related to the issue fixed: #22 

PS: This is my very first open source contribution so feel free to correct me if I did something wrong.

### CODE REVIEW

File: l10n/po/pt_BR/_guides/redis.adoc.po [hunk 1]
```diff
@@ -181,40 +174,39 @@ msgstr ""
...
+msgstr "Criar o arquivo `src/main/java/org/acme/redis/IncrementoService.java`, com o seguinte conteúdo:"
 
 #. type: delimited block -
 #: upstream/_guides/redis.adoc:109
-#, no-wrap
 msgid ""
 "import jakarta.enterprise.context.ApplicationScoped;\n"
 "import jakarta.inject.Inject;\n"
 msgstr ""
+"import jakarta.enterprise.context.ApplicationScoped;\n"
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/redis.adoc.po:205]: No need to translate when it's code or the same content, but that' okay

**Review by gastaldi — APPROVED** [review]
We usually avoid translating code because if the code changes, you don't need to rewrite it again on every translation. 

But I realize you have made a tremendous effort to get that translated, so let's merge that and avoid changing code in the future

**Comment by judalabs** [comment]
Thanks for the quick reply! I have a few comments:

1 I didn't understand why the build failed. Did I do something wrong?

2 good to know that it's not necessary to translate the code. Maybe we could add that to the tutorial on discord

File: l10n/po/pt_BR/_guides/redis.adoc.po [hunk 2]
```diff
@@ -708,26 +831,34 @@ msgid ""
...
 #: upstream/_guides/redis.adoc:382
 msgid "If you followed the instructions, you should have the Redis server running.  Then, you just need to run the application using:"
-msgstr "Se seguiu as instruções, deve ter o servidor Redis funcionando. Depois, você só precisa executar a aplicação utilizando:"
+msgstr "Se você seguiu as instruções, deve ter o servidor Redis funcionando. Depois, você só precisa executar a aplicação utilizando:"
 
 #. type: Plain text
 #: upstream/_guides/redis.adoc:386
 msgid "Open another terminal and run the `curl http://localhost:8080/increments` command."
-msgstr "Abra outro terminal e execute o comando `curl http://localhost:8080/increments."
+msgstr "Abra outro terminal e execute o comando `curl http://localhost:8080/incrementos."
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/redis.adoc.po:858]:
```suggestion
msgstr "Abra outro terminal e execute o comando `curl http://localhost:8080/incrementos`."
```

File: l10n/po/pt_BR/_guides/redis.adoc.po [hunk 3]
```diff
@@ -899,9 +1025,8 @@ msgstr "Vamos iniciar um servidor Redis na porta 6379 usando:"
 
 #. type: delimited block -
 #: upstream/_guides/redis.adoc:483
-#, no-wrap
 msgid "docker run --ulimit memlock=-1:-1 -it --rm=true --memory-swappiness=0 --name redis_quarkus_test -p 6379:6379 redis:5.0.6\n"
-msgstr ""
+msgstr "docker run --ulimit memlock=-1:-1 -it --rm=true --memory-swappiness=0 --name redis_quarkus_test -p 6379:6379 redis:5.0.6\n"
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/redis.adoc.po:1029]:
```suggestion
msgstr ""
```

File: l10n/po/pt_BR/_guides/redis.adoc.po [hunk 4]
```diff
@@ -938,17 +1061,15 @@ msgstr "Este comando iniciará uma instância Redis para executar os testes."
...
 msgid "Then run it:"
-msgstr "Em seguida, execute-o:"
+msgstr "Em seguida, execute:"
 
 #. type: delimited block -
 #: upstream/_guides/redis.adoc:507
-#, no-wrap
 msgid "java -jar target/quarkus-app/quarkus-run.jar\n"
-msgstr ""
+msgstr "java -jar target/quarkus-app/quarkus-run.jar\n"
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/redis.adoc.po:1069]:
```suggestion
msgstr ""
```

File: l10n/po/pt_BR/_guides/redis.adoc.po [hunk 5]
```diff
@@ -938,17 +1061,15 @@ msgstr "Este comando iniciará uma instância Redis para executar os testes."
...
 
 #. type: Title ==
 #: upstream/_guides/redis.adoc:509
-#, no-wrap
 msgid "Running Native"
 msgstr "Funcionamento Nativo"
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/redis.adoc.po:1074]:
```suggestion
msgstr "Executando em modo nativo"
```

File: l10n/po/pt_BR/_guides/redis.adoc.po [hunk 6]
```diff
@@ -969,13 +1090,11 @@ msgstr "Quando a compilação estiver concluída, pode rodar o executável com:"
 
 #. type: delimited block -
 #: upstream/_guides/redis.adoc:527
-#, no-wrap
 msgid "./target/redis-quickstart-1.0.0-SNAPSHOT-runner\n"
-msgstr ""
+msgstr "./target/redis-quickstart-1.0.0-SNAPSHOT-runner\n"
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/redis.adoc.po:1094]:
```suggestion
msgstr ""
```

**Comment by gastaldi** [comment]
https://pr-preview-27-pt-quarkusio.surge.sh/guides/redis

**Review by gastaldi — APPROVED** [review]

**Pull request merged**